manifests/l/Lauriethefish/QuestPatcher/2.2.0/Lauriethefish.QuestPatcher.installer.yaml

```
@@ -0,0 +1,23 @@
+﻿# Created using YamlCreate.ps1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Lauriethefish.QuestPatcher
+PackageVersion: 2.2.0
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: inno
+#  Scope: 
+  InstallerUrl: https://github.com/Lauriethefish/QuestPatcher/releases/download/2.2.0/QuestPatcher-windows.exe
+  InstallerSha256: 6B679A74E620C49E05077A7F430EB407336A9E137EEC8E72A4CC58037735E60B
+  InstallerSwitches:
+    Silent: /VERYSILENT /ALLUSERS
+    SilentWithProgress: /SILENT
+  UpgradeBehavior: install
```

Let's add user scope for compatibility reasons.

```suggestion
- InstallerLocale: en-US
  Architecture: x64
  InstallerType: inno
  Scope: machine
  InstallerUrl: https://github.com/Lauriethefish/QuestPatcher/releases/download/2.2.0/QuestPatcher-windows.exe
  InstallerSha256: 6B679A74E620C49E05077A7F430EB407336A9E137EEC8E72A4CC58037735E60B
  InstallerSwitches:
    Silent: /VERYSILENT /ALLUSERS
    SilentWithProgress: /SILENT /ALLUSERS
  UpgradeBehavior: install
- InstallerLocale: en-US
  Architecture: x64
  InstallerType: inno
  Scope: user
  InstallerUrl: https://github.com/Lauriethefish/QuestPatcher/releases/download/2.2.0/QuestPatcher-windows.exe
  InstallerSha256: 6B679A74E620C49E05077A7F430EB407336A9E137EEC8E72A4CC58037735E60B
  InstallerSwitches:
    Silent: /VERYSILENT /CURRENTUSER
    SilentWithProgress: /SILENT /CURRENTUSER
```